tests/manual/assets/scripts.js

```diff
@@ -120,7 +108,7 @@
         let property = parts.shift();
 
         if (path === property) {
-            object[property] = value === 'Infinity' ? Infinity : value;
+            object[property] = value;
```

Comment:
I guess that works because of automatic type conversion?

Reply:
it isn't needed anymore - we moved the "unlimited" indicator to the "flag" property